src/infra/auth/authCmdRepo_test.go

```diff
@@ -10,13 +10,18 @@ import (
 
 func TestAuthCmdRepo(t *testing.T) {
 	testHelpers.LoadEnvVars()
+	authCmdRepo := AuthCmdRepo{}
 
 	t.Run("GetSessionToken", func(t *testing.T) {
-		authCmdRepo := AuthCmdRepo{}
+		ipAddress, err := valueObject.NewIpAddress("127.0.0.1")
```

Comment:
Use the NewLocalhostIpAddress constructor in these cases.

Reply:
Done.